Source/Math/NcclComm.cpp

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
```

Comment:
Eldar: let's maybe talk tomorrow about this. There's no separate dir for MathCUDA, so the dir would be appropriate. The file however also contains the stub (lines 99 and below). On the other, there's a dependency on MPI (via the header), which normally MathCUDA / Math doesn't have. Maybe we can take a couple of minutes to see how we could untangle this.

---
In reply to: [87001650](https://github.com/Microsoft/CNTK/pull/874#discussion_r87001650) [](ancestors = 87001650)